docs/file_handling.md

```diff
@@ -8,11 +13,30 @@ Image developers may run the root container and point to a Nulecule directory on
 
 ## Directories
 
-* `/tmp/<atomicapp_name>`: Host directory for temporary Nulecule files. May be overridden.
-* `/tmp/<atomicapp_name>/.workdir`: Host directory for artifact template files with variable substitution.
+* `external/`: External atomic apps, if any, for the given atomic app are
```

Comment:
Might be good to move this to below the next entry and change the name to:

`/var/lib/atomicapp/<atomic app name>-<uuid>/external`